proofs/fstar/extraction/Libcrux.Kem.Kyber768.Arithmetic.fst

```diff
@@ -0,0 +1,145 @@
+module Libcrux.Kem.Kyber768.Arithmetic
```

Comment:
What is the difference between this and the `formal_verification` directory? Also should these files be under a `kyber768` sub-directory in `proofs`?

Reply:
There is no relation yet.
The `formal_verification` dir is a manual thing about all the other code. This is the first piece of code that will be proven here in libcrux directly. And we will have to figure out how to structure that.
There is work on hax (e.g. https://github.com/hacspec/hacspec-v2/pull/202) to work towards "verifiable crates" that define how to integrate the proofs and where this code lives.

For now we do what hax is doing, that is putting the extracted code into the `proofs` directory. Note that this is an extraction of the libcrux crate, not kyber.